.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,62 @@
+<!--
+  Make sure you have read CONTRIBUTING.md completely before you file a new
+  issue! 
+
+  If possible, try to determine if the bug is actually part of the Swift driver,
+  or if the issue is actually from `libmongoc` or `libbson`. If so, you should
+  file the issue with the representative projects.
+-->
+
+## Information
+
+### Swift version
+
+What does this command give you?
+```
+$ swift --version
+```
+
+### Operating system
+
+What does this command give you?
+```
+$ uname -a
+```
+
+### Driver version
+
+What does this command give you?
+```
+$ cat Package.swift # Only applies if you are using Swift package manager
+```
+
+### Version of `libmongoc` and `libbson`
+
+What does this command give you?
+```
+$ brew list --versions mongo-c-driver # Only applies if you installed via brew
+```
```

Comment:
are there ways people can check their version if they installed it in another way? if so, let's include them.

Reply:
No idea. If they installed it manually, there is no real universal way. I am not familiar with other methods of installation. Are you?

Reply:
Seems like from the recent issue (#141) posted to the repo that pods install things like `libmongoc` locally somehow.

Reply:
You can check via `pkg-config`, `brew` and `apt` - these are all supported via SwiftPM. So you'll want to include

Reply:
Alternatively, the `mongoc-version.h` and `bson-version.h` files in `/usr/include` (or equivalent) contain relevant information, but I don't think we want to suggest that users go poking around in there.

Reply:
@jmikola I pretty much refrained from that as well because users can technically have such files _anywhere_ as long as their build configuration is aware of it. `pkg-config` is a little better, since it is supposed to manage these things, but even `pkg-config` can omit details if the build configuration the user is using explicitly specifies the location of where to link with the C libraries (anyone: correct me if I'm wrong).

In either case though, most people, myself included, will rely on `pkg-config` so it seems 'ok' to suggest using that. `apt` of course is okay int the same way `brew` is.

Reply:
I've added a bit for `apt`. However, `pkg-config` doesn't seem to have much going for it besides `-cflags`. Does anyone know what we should be looking for in the output from `-cflags`. Perhaps @jmikola, if you use it on Linux?

Reply:
does --modversion not work?

Reply:
Yup, sorry, `--modversion` works for this too. But what I don't know is what the output of that looks like. Essentially, I would like to make a complete command that `grep`'s out the libraries we're looking for (`libmongoc` and `libbson`).

E.g., just want someone to confirm that something like:
```
$ pkg-config --modversion | grep -E "(libmongoc|libbson)"
```
Actually does the trick, or if the regex should be different.

Reply:
```
$ pkg-config --modversion libmongoc-1.0
1.13.1-dev

$ pkg-config --modversion libbson-1.0
1.13.1-dev
```

Reply:
Thanks! Done.

Reply:
BTW: I think this hits on all the options we want to support, so I think we can resolve this.